pre_commit_hooks/trailing_whitespace_fixer.py

```diff
@@ -7,10 +7,15 @@
 from typing import Sequence
 
 
-def _fix_file(filename, is_markdown):  # type: (str, bool) -> bool
+def _fix_file(filename, is_markdown, chars_to_strip):
```

Comment:
I would maybe just call this argument `chars` to match the commandline option -- you'll need to reflow less too :+1:

Reply:
Addressed in commit a33a8f0.